crates/cosmos/cosmos-chain-components/src/impls/queries/eip/dispatch.rs

```diff
@@ -0,0 +1,37 @@
+use cgp::core::error::CanRaiseError;
+use prost::DecodeError;
+use reqwest::Response;
+
+use crate::impls::queries::eip::feemarket::QueryEipFromFeeMarket;
+use crate::impls::queries::eip::osmosis::OsmosisQueryEip;
+use crate::traits::eip::eip_query::EipQuerier;
+use crate::traits::eip::eip_type::{EipQueryType, HasEipQueryType};
+use crate::traits::rpc_client::HasRpcClient;
+
+#[derive(Debug)]
+pub struct EipQueryError {
+    pub response: Response,
+}
+
+pub struct DispatchQueryEip;
+
+impl<Chain> EipQuerier<Chain> for DispatchQueryEip
+where
+    Chain: HasRpcClient
+        + HasEipQueryType
+        + CanRaiseError<reqwest::Error>
+        + CanRaiseError<subtle_encoding::Error>
+        + CanRaiseError<DecodeError>
+        + CanRaiseError<core::num::ParseIntError>
+        + CanRaiseError<core::num::ParseFloatError>
+        + CanRaiseError<EipQueryError>,
```

Comment:
The trait bounds here can be simplified to:

```
    QueryEipFromFeeMarket: EipQuerier<Chain>,
    OsmosisQueryEip: EipQuerier<Chain>,
```

Reply:
Done in a23a8cb55096d3857c99f73ae503b8c54c176a0a